docs/example_static_html.md

```diff
@@ -169,10 +180,24 @@ Tilt was able to update the container in less than a second!
 
 ### Final Score
 
-| Approach | Deploy Time |
-|---|---|
-| Naive | 1-2s |
-| With live_update | 0-1s |
+<table class="benchmark-report">
```

Comment:
there's a much easier way to do this, just prepend `{:.benchmark-report}` before the markdown table

Reply:
alternatively, just change the css to `table:not(.docutils)`

Reply:
😅will do